ods_ci/tests/Resources/Files/llm/serving_runtimes/caikit_tgis_servingruntime_grpc.yaml

```diff
@@ -10,18 +10,48 @@ spec:
       name: caikit
   containers:
     - name: kserve-container
-      image: quay.io/modh/text-generation-inference@sha256:a17a2868644929ee844ceb2778ac3f6db0936824d9b89d11ea7aa059466fcd0b
+      image: quay.io/modh/text-generation-inference@sha256:9f18a63cd84b084c3cbf15534f22d5ba6916d9501298abcc03271d26ebf5cdfb
       command: ["text-generation-launcher"]
-      args: ["--model-name=/mnt/models/artifacts/"] 
+      args: ["--model-name=/mnt/models/artifacts/"]
       env:
         - name: TRANSFORMERS_CACHE
           value: /tmp/transformers_cache
+      # resources: # configure as required
+      #   requests:
+      #     cpu: 8
+      #     memory: 16Gi
```

Comment:
Are these two commented blocks for resources in these two files necessary to keep?